Expensify/expensify-common
# Add alwaysShowSelectedOnTop, pass children when searching

@tgolen , will you please review this?

Needed for https://github.com/Expensify/Expensify/issues/71562

# Tests
1. Check that children are passed after searching (keeping the style in the attendees selector, showing the avatar).
1. Check that you can select (by clicking) the `Add X` option in the attendee selector.
1. Check that selected options are shown on the top of the list, only when the prop is passed.
1.  Make sure we show the "Add X" even when what you searched matches the items in the list.
1. Test other components using the combobox.

# QA
1. No, hax will qa the whole feature.
